lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
```

Comment:
I'd like to avoid having to write this function, if possible.  JSON supports lists/arrays just fine, so wouldn't `JSON.stringify` just work?